jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -19,6 +19,14 @@
                 persistent-hint
               ></v-switch>
             </v-row>
+            <v-row>
+              <v-switch v-if="config=='specviz'"

```

Comment:
cubeviz and mosviz also have spectral viewers, so I think we want the switch there as well.

```suggestion
              <v-switch v-if="['specviz', 'cubeviz', 'mosviz'].indexOf(config) !== -1"

```